ios/MessageWebViewWrapper.swift

```diff
@@ -3,6 +3,7 @@
 import Foundation
 import AirshipKit
 import AirshipFrameworkProxy
+import WebKit
```

Comment:
Do you need WebKit?

Reply:
It references WKWebView within it so I think so. I dont know why it was working before